content/800-guides/240-partner-api.mdx

```diff
@@ -0,0 +1,163 @@
+
+For a complete list of available endpoints and details on request/response formats, see the [Prisma Management API documentation](/postgres/database/api-reference/management-api).
+
+## Provisioning a Database as a Partner
+
+To provision a new Prisma Postgres database for your users as a partner, follow these steps:
+
+1. **Collect user input**: Gather the necessary information from your user, such as the desired region, database name, and any other options your app supports.
+2. **Authenticate your integration**: Use your integration token (obtained via OAuth2) to authenticate API requests. This token proves your app is an approved partner.
+3. **Send a provisioning request**: Make a `POST` request to the Partner API endpoint to create a new project (database). For example:
+
+```ts
+const prismaResponse = await fetch('https://api.prisma.io/v1/projects', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+    Authorization: `Bearer <YOUR_INTEGRATION_TOKEN>`,
+  },
+  body: JSON.stringify({ region, name }),
+});
+```
+
+4. **Handle the response**: If successful, the API will return the new project's details, including connection strings and a `project_id`. Store these securely and display them to your user as needed.
+
+5. **(Optional) Store project metadata**: You may want to associate the `project_id` with your user in your own database for future reference.
+
+## Database Claim Flow
+
+Once a database is provisioned, you may want to transfer ownership to your user so they can manage it in their own Prisma workspace. This is done via the claim flow, which consists of three main steps:
+
+### Overview: How the Claim Flow Works
+
+When a user wants to claim a database, your app will:
+
+1. Trigger the OAuth2 flow, redirecting the user to Prisma Auth.
+2. The user authenticates and selects a workspace.
+3. Your backend receives an authorization code, exchanges it for a user access token, and calls the Partner API transfer endpoint with both your integration token and the user's token.
+
+This ensures the transfer is secure and only the intended user can claim the database.
+
+```mermaid
+flowchart TD
+  A[User clicks 'Claim Database'] --> B[Redirect to Prisma Auth (OAuth2)]
+  B --> C[User authenticates & selects workspace]
+  C --> D[Prisma redirects to your callback with code & state]
+  D --> E[Your backend exchanges code for user access token]
+  E --> F[Your backend calls Partner API transfer endpoint]
+  F --> G[Database ownership transferred to user workspace]
+```
+
+### 1. Triggering the Claim Flow
+
+When your user is ready to claim a database, provide a button or link in your app (e.g., "Claim Database"). When clicked, your backend or serverless function should:
+
+- Generate a secure `state` value to track the session.
```

Comment:
I'm not sure what we mean here, likely needs more context/description or a different way to explain it